test/unit/org/apache/cassandra/db/virtual/SettingsTableTest.java

```diff
@@ -154,7 +155,7 @@ public void testEncryptionOverride() throws Throwable
         check(pre + "protocol", "[TLSv5]");
 
         config.server_encryption_options = config.server_encryption_options.withProtocol("TLS");
-        check(pre + "protocol", "[TLSv1.3, TLSv1.2, TLSv1.1, TLSv1]");
+        check(pre + "protocol", "[" + String.join(", ", SSLFactory.tlsInstanceProtocolSubstitution()) + "]");
```

Comment:
isn't this simpler?

```
check(pre + "protocol", SSLFactory.tlsInstanceProtocolSubstitution()).toString());
```

Reply:
Agreed, didn't think to try the default formatting.